src/tools/bzlmod/utils.bzl

```diff
@@ -36,6 +36,14 @@ def extract_url(attributes):
     else:
         fail("Could not find url in attributes %s" % attributes)
 
+_WELL_KNOWN_MODULES = ["bazel_tools", "local_config_platform", "platforms"]
```

Comment:
Is this file going to be replaced with `bazel vendor` soon? I can flesh out and document the new logic more, but ideally it won't even be needed anymore with that new feature.